AzureFunctions.Autofac.Shared/Provider/Config/InjectExtensionConfigProvider.cs

```diff
@@ -8,33 +8,34 @@
 using Microsoft.Azure.WebJobs.Host.Bindings;
 
 #if NETSTANDARD2_0
+using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
 #endif
 
 namespace AzureFunctions.Autofac.Provider.Config
 {
     public class InjectExtensionConfigProvider : IExtensionConfigProvider
     {
-        private readonly InjectBindingProvider bindingProvider;
-        
-        #if NET46
-        public InjectExtensionConfigProvider()
-        {
-            this.bindingProvider = new InjectBindingProvider();
-        }
-        #endif
-        
-        #if NETSTANDARD2_0
-        public InjectExtensionConfigProvider(IOptions<ExecutionContextOptions> options)
+        private InjectBindingProvider _bindingProvider;
+
+#if NET46
+        public InjectExtensionConfigProvider() { }
+#endif
+
+#if NETSTANDARD2_0
+        public InjectExtensionConfigProvider(IOptions<ExecutionContextOptions> options, ILoggerFactory loggerFactory)
         {
             var appDirectory = options.Value.AppDirectory;
```

Comment:
I'm not entirely sure why I have this as a local variable. I am going to investigate that.